js_modules/dagster-ui/packages/ui-core/src/sensors/SensorTargetList.tsx

```diff
@@ -82,12 +84,19 @@ const AssetSelectionLink = ({
   const [showAssetSelection, setShowAssetSelection] = React.useState(false);
 
   const sortedAssets = React.useMemo(() => {
-    return assetSelection.assets
+    if (assetSelection.assetsOrError.__typename === 'PythonError') {
+      showCustomAlert({
+        title: 'Error',
+        body: <PythonErrorInfo error={assetSelection.assetsOrError} />,
+      });
```

Comment:
am i interpreting this correctly that it pops up an error dialog as soon as the page loads? If so that might be a bit jarring, I was imagining like a "View error" link that you click on?

Reply:
Ah okay yeah I can do it that way